test/e2e/memcached_test.go

```diff
@@ -16,6 +16,8 @@ package e2e
 
 import (
 	"bytes"
+	goctx "context"
```

Comment:
nit: Any reason why rename this to goctx?

Reply:
Ah, that was just to avoid a collision that is no longer is a problem. I'll change it back.